operator/v1/types_ingress.go

```diff
@@ -376,6 +381,15 @@ type LoadBalancerStrategy struct {
 	// +required
 	Scope LoadBalancerScope `json:"scope"`
 
+	// allowedSourceRanges specifies an allowlist of IP address ranges to which
+	// access to the load balancer should be restricted.  Each range must be
+	// specified using CIDR notation (e.g., "10.0.0.0/8").  If no range is
+	// specified, "0.0.0.0/0" is used by default, which allows all source addresses.
```

Comment:
IPv6 is allowed by default, right?  So the default is really two entries: "0.0.0.0/0" for IPv4 and "::/0" for IPv6.

Reply:
I changed the comment.